reference/03-Blocks/unregister-block.md

```diff
@@ -0,0 +1,54 @@
+---
+sidebar_position: 10
+---
+
+# Unregister a Block
+
+## Reference Guide WordPress Documentation:
+* [unregisterBlockType](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-blocks/#unregisterblocktype)
+
+## Use Case:
+Gutenberg comes with a lot of core blocks. When creating custom builds for clients all of these core blocks are generally not needed nor should they be used by the client. In these situations it is necessary to disable the blocks from the block editor so that a user can not access them when creating new pages and posts with the block editor. This can be done using the `unregisterBlockType` function and passing in the blocks which should be disallowed within a `.js` file. 
+
+
+## Code Example:
+This example showcases how an array of blocks can be passed into a single function and looped through to disable multiple blocks at the same time. 
+
+`block-filters/unregister-blocks.js`
+```bash
+const { unregisterBlockType } = wp.blocks;
+
+const disallowedBlocks = [
+	'core/preformatted',
+	'core/table',
+	'core/code',
+	'core/video',
+	'core/verse',
+	'core/avatar',
+	'core/audio',
+	'core/calendar',
+	'core/freeform',
+];
+
+wp.domReady(() => {
+	wp.blocks.getBlockTypes().forEach(function (blockType) {
+		if (disallowedBlocks.indexOf(blockType.name) !== -1) unregisterBlockType(blockType.name);
+	});
```

Comment:
I reworked the code example slightly to be a little easier to parse. Also as per our coding standards we tend to avoid inline control statements because they can easily cause issues when refactoring :) 

So 

```js
// instead of this
if ( condition ) doSomething();

// we always want to write out the full if statement
if ( condition ) {
    doSomething();
}
```
 
```suggestion
	const blocks = getBlockTypes();
	blocks.forEach(({ name }) {
		if (disallowedBlocks.includes(name)) {
			unregisterBlockType(name);
		}
	});
```